index.html

```diff
@@ -50,7 +51,143 @@ <h2 class=" role"> I am a Software developer</h2>
     </section>
     
 </div>
+    <div class ="worksection">
+        <section id = "worksectionheader">
+        <h1 id= "worksectionheading"> My Recent Works</h1>
+        <img id="sectiondivider" src ="image/Vector (10).png">
+        </section>
+        
+        <section class="workdescription" >
+            <div id ="project1">
+                <img id ="placeholder" src="image/Img Placeholder.png">
+                <h2 id="project1text">Multi-Post Stories</h2>
+                <p id="project1supporttext">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul id ="project1language">
+                    <li class ="languageitem">css</li>
+                    <li class ="languageitem">html</li>
+                    <li class ="languageitem" id ="bootstrap">bootstrap</li>
+                    <li class ="languageitem">Ruby</li>
+                </ul>
+                <button id="seeproject1">See Project</button>
+            </div>
```

Comment:
- [ ] Your design is different from the provided Figma design in many ways. Please check for all margins and heights. I see you use a lot of fixed heights (view height). You should not do that for many reasons, but most importantly, allow the height of your element to grow naturally with its contents unless you truly need a fixed height. In the case of this project cards. you should allow your cards to have responsive height.

Figma Design | Your Design
---- | ----
![image](https://user-images.githubusercontent.com/84156714/212473208-b4c2febb-55b1-49b8-93df-d69146843fa9.png) | ![image](https://user-images.githubusercontent.com/84156714/212473241-5b21b582-ff7a-4d41-a92a-7f19a762855f.png)